cub/cub/thread/thread_operators.cuh

```diff
@@ -50,6 +50,7 @@
 #include <cub/detail/type_traits.cuh> // always_false
 #include <cub/util_type.cuh>
 
+#include <cuda/__functional/address_stability.h>
 #include <cuda/functional>
```

Comment:
I believe we should not need to add a special include, given that we opted to use the main headers for cub / thrust